runtime/j9vm/j9vmnatives.xml

```diff
@@ -329,6 +332,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<export name="JVM_GetNestHost"/>
 		<export name="JVM_GetNestMembers"/>
 		<export name="JVM_AreNestMates"/>
+		<export name="JVM_InitClassName">
+			<exclude-if condition="spec.java12 and not spec.java13"/>
```

Comment:
This doesn't seem right, it should be excluded for java8 and java12

Reply:
This group, `jdk11`, is only used for Java 11+ (which excludes it for Java 8).

See my comment above: The condition excludes the export for Java 12+, but not Java 13+ (which means only for Java 12).